### ISSUE TITLE
Multiple Servers

### ISSUE THREAD

**Issue by kkranzo** [Automattic/wp-super-cache#115]
I know I have seen various threads that recommend using Batcache if you have mutiple servers. However, our site scales up and down and most of the time only has one server. Also, I like how some of the other plugins we use correspond with WP Super Cache. So, my questions is... is there any reason why the below setup wouldn't work?

- 1 Master Web Server
- Several Slave Web Servers
- All sharing the same Database
- All servers have the WP Super Cache cache folder on each server
- All WP Admin changes happen on Master server
- Any changes in cache files get Lsynced over to all slave servers

So, basically all servers use the cache files residing on their own server. If the cache gets cleared on the Master, then it will then sync and clear on the Slave servers.

This should be fine, correct?

**Comment by kraftbj** [comment]
That... should... be fine, though not something we've tested against :-). Any requests on the slave servers that would generate cached files—are those sync'd anywhere or only benefit that particular slave?

That would be the biggest pitfall I'd see if someone is hitting an uncached page on a slave server, that hit might not benefit the rest of the infrastructure.

**Comment by kkranzo** [comment]
Gotcha @kraftbj.

Yeah, the requests on the slave servers would generate cached files that would just benefit that particular slave. So we should probably set it up to sync the /cache directory between the servers then so that they would all share the same static files, correct?

**Comment by kkranzo** [comment]
We will set it up so the caching basically happens from the Master. So it would sync it's cache directory out to the slave servers. And that way, when the cache is cleared on the master (wp-admin happens on the master), then the cache would get cleared on the slaves as well.

**Comment by kraftbj** [comment]
That sounds good. I think the potential headaches of having slaves sync caches wouldn't be worth it all-in-all. Having all canonical changes on the master is probably the way I'd would go given using WPSC on multiple servers.

**Comment by kkranzo** [comment]
@kraftbj - Thanks for the input!

**Comment by cgidds** [comment]
@kraftbj Sorry to respond to a closed issue, but it's related to the topic. 

Has there been much success in running WP Super Cache in AWS's beanstalk environment, which doesn't have a master-slave relationship but individual/separate servers delivering content? We too could setup a shared file system similar to what @kkranzo, but that could be more of a maintenance nightmare.
